mchf-eclipse/drivers/ui/ui_driver.c

```diff
@@ -6370,6 +6371,10 @@ void UiDriver_MainHandler()
 				{
 					UiDriver_UpdateLcdFreq(df.tune_old/TUNE_MULT, Yellow, UFM_SECONDARY);
 				}
+				else if (ts.dmod_mode == DEMOD_CW)
+				{
+					UiDriver_UpdateLcdFreq(ads.snap_carrier_freq, Green, UFM_SECONDARY);
+				}
```

Comment:
I think you are not going to make friends with the RIT users here, I believe RIT is being used in CW.  And I also think, that use of RIT will break the correct frequency display anyway as the code adds the RIT offset to the passed frequency. Not good style but this is how it is coded.

Reply:
Well, I do not see a problem here: RIT users can switch off this option ;-). Additionally, the currrent green frequency display is just to show that carrier frequency estimation is possible in this first alpha version. The display will be no longer useful once the graphical offset indicator is ready.